NFC blog (#68)
xmartlabs/blog: _posts/2018-11-27-how-NFC-can-help-improve-your-business.md

Paypal Visa and Master Card has follow this same model and create applications that allows the user to pay with its cards or accounts from its smarthphones.

NFC based payment methods have been accepted for a widely variety of stores banks and cards providers around the world.

### Rapid and effective data authentication

NFC cards are already being use to identify most of the people in the world, since the big majority of the world passports have an NFC chip that contains the person information. Biometric information is usually storage in this type of chips allowing to identify its owner and provide other critical information like its digital signature. Since the creation of the standard a lot of countries have transitioned to the electronic passport in order to make their identifications processes faster and more efficients.

<img width="100%" src="/images/how-nfc-can-help-improve-your-business/linked-in.jpg" /> 
<br />
<p style="text-align: center;"> (source: <a href="https://www.instructables.com/">Instructables</a>)</p>

Some companies decide to equip theirs business card with this technology in order to identify its employees and show its resumes.

> Banking, governmental agencies and other enterprises have been benefit from this technology if you want to know how this could help your enterprise workflows and make it more efficients, [contact us today.](#JotFormIFrame-82355834626665) 

Reviewer: ❤️

Reviewer: And change to send an email.

<iframe width="100%" height="383px" src="https://youtube.com/embed/Zbxc1FxqHP4" frameborder="0" allowfullscreen></iframe>

Some companies have come up with lockers that require both a NFC key that contains the biometric information of the person that is trying to open it so in order to open a lock it has a two factor authentication process like system.

<img width="100%" src="/images/how-nfc-can-help-improve-your-business/nfc-key.jpg" /> 
<br />
<p style="text-align: center;"> (source: <a href="https://www.theverge.com">The Verge</a>)</p>

In some events NFC cars where used to give easily access to certain areas and control the identity of the attendees. 

> To learn how NFC keys can help your business improve access and identification, feel free to [contact us with your questions.](#JotFormIFrame-82355834626665) 

Reviewer: Same here

<img width="100%" src="/images/how-nfc-can-help-improve-your-business/skylanders-items.jpg" /> 
<br />
<p style="text-align: center;"> (source: <a href=https://www.theverge.com">The Verge</a>)</p>

There are also a variety of consoles that have NFC integrated to them. In this cases some developers have come up with applications that could link devices to exchange game objects or share game play adding to it a competitive and cooperative experience. 

This kinds of interactions help make the user experience more remarkable to the user and help make user interact more with the game itself. 

### Interested in building NFC based applications get in touch!

A way to exchange information pair to pair securely and efficient, NFC based applications allows you to create mechanism to not only make yours business presses more secure but to improve their efficiency and help create a more appeal user experience. This technology is used in a variety of industries and have prove to provide an improvement to its owners.

> To learn more about how NFC based apps can make improve your business, [reach out to us today.](#JotFormIFrame-82355834626665)

Reviewer: Change this to be a mail to hi@xmartlabs.com

> <iframe

Reviewer: remove this :)

---
layout: post
title:  How NFC can help improve your business
date: 2018-10-08 09:00:00
author: XL Team
categories: AR
author_id: xl

---

In 2017 Apple made IOS 11 support NFC application with this move apples joins Google and others device manufacturers that provides SDK's to integrate this technology. This action has widely increase the amount of devices with custom NFC applications available in the market. 

> NFC its a communication protocol that enable two devices to share data withing a range of 4 cm that was launched to the public in 2004. This technology is usually used to share personal or critical data between two devices, since its range restrictions make it more secure than Bluetooth. In this post we will explore some use cases that have display some of the benefits of this technology.

Author: sounds good to me

### Quickly and effortless payments

NFC its not a new thing when we speak about payments, since 2014 Apple, Google and Samsung have been working in a Wallet of their own that allow you to pay through a NFC POS terminal. This wallets prove to be very secure and convenient since the number of smartphones and NFC POS terminals have been increasing in the past years added to the ability to pay however you want without having to take your cash, credit/debit card or checkbook out of your home.

<iframe width="100%" height="383px" src="https://www.youtube.com/embed/bXbgtpkkTqw" frameborder="0" allowfullscreen></iframe>

> Paypal Visa and Master Card has follow this same model and create applications that allows the user to pay with its cards or accounts from its smarthphones.

Reviewer 2: `Paypal, Visa and Mastercard have also followed this same model to create applications that allow the user to pay with their credit or debit cards as well as their bank accounts directly from their smartphone, making use of NFC.`

> NFC based payment methods have been accepted for a widely variety of stores banks and cards providers around the world.

Reviewer 2: `NFC-based`

Reviewer 2: `...in a wide variety of stores, banks and other entities all around the world.`

### Rapid and effective data authentication

> NFC cards are already being use to identify most of the people in the world, since the big majority of the world passports have an NFC chip that contains the person information. Biometric information is usually storage in this type of chips allowing to identify its owner and provide other critical information like its digital signature. Since the creation of the standard a lot of countries have transitioned to the electronic passport in order to make their identifications processes faster and more efficients.

Reviewer 2: `NFC chips` instead of `NFC cards`?

Reviewer 2: `...are already being used...`, `...since the big majority of the passports have a chip like this embedded, containing information related to each person.`

Reviewer 2: `Biometric information is usually stored in these kind of chips too, allowing to identify its owner and also provide access to other features like a digital signature. Since the creation of the standard, a lot of countries have transitioned to these electronic passports in order to make their identifications processes faster and more efficient.`

<img width="100%" src="/images/how-nfc-can-help-improve-your-business/linked-in.jpg" /> 
<br />
<p style="text-align: center;"> (source: <a href="https://www.instructables.com/">Instructables</a>)</p>

> Some companies decide to equip theirs business card with this technology in order to identify its employees and show its resumes.

Reviewer 2: `In addition, some companies have decided to equip their business cards with this technology, providing an easy way to identify their employees and showing their resumes.`

> Banking, governmental agencies and other enterprises have been benefit from this technology if you want to know how this could help your enterprise workflows and make it more efficients, <a href="mailto:hi@xmartlabs.com">contact us today</a>.

Reviewer 2: `Financial and government agencies, as well as other companies have benefited from this technology. If you want to know how this could help your company's workflow, making them more efficient, <a href="mailto:hi@xmartlabs.com">contact us today</a>.`

> ### Safe and efficient way to give access to yours employees or invitees 

Reviewer 2: `...to your employees...`

> NFC keys are objects that allows you to lock and unlock doors or any type of object that is close. This keys are compose of two parts the actual key and a locker. Since it operates under the NFC protocol it is necessary to hold the key in a certain distance to the object you are trying to unlock. Only the key contains the data to unlock it made this a very secure and convenient method. This kind of keys are used by AirBnb owners or enterprises because it is easy to reconfigure and to give access to theirs employees or invitees. 

Reviewer 2: `...that allow you to lock...`, `...or any other object with an NFC-enabled lock close by.`

Reviewer 2: `These keys are composed of two elements: the key and the lock.`

Reviewer 2: `...to hold the key to a certain distance...`, `...you are trying to lock or unlock.`?

Reviewer 2: `...making this a very secure and...`

Reviewer 2: `These kind of keys are used by many AirBnb owners and other enterprises because it's easy to reconfigure, giving access to their guests or employees.`

<iframe width="100%" height="383px" src="https://youtube.com/embed/Zbxc1FxqHP4" frameborder="0" allowfullscreen></iframe>

> Some companies have come up with lockers that require both a NFC key that contains the biometric information of the person that is trying to open it so in order to open a lock it has a two factor authentication process like system.

Reviewer 2: which are the both things? I only see one, maybe something is missing?

<img width="100%" src="/images/how-nfc-can-help-improve-your-business/nfc-key.jpg" /> 
<br />
<p style="text-align: center;"> (source: <a href="https://www.theverge.com">The Verge</a>)</p>

> In some events NFC cars where used to give easily access to certain areas and control the identity of the attendees. 

Reviewer 2: `...NFC cards were given, allowing attendees to have easy access to certain areas.`

> To learn how NFC keys can help your business improve access and identification, feel free to <a href="mailto:hi@xmartlabs.com">contact us with your questions</a>.

Reviewer 2: `To learn how NFC keys can help your business improve access control and identification...`

Reviewer 2: `...contact us with any questions...`?

### Easy and enjoyable interactions

> Skylanders its a game that uses NFC interaction between devices to get the user experience to the next level. Adding NFC chips to figures and making the game more appeal to kids and families. 

Reviewer 2: `it's`

Reviewer 2: `to the next level, adding NFC chips to action figures, making the game more appealing to its users.`

> Skylanders it's a game that uses NFC interaction between devices to get the user experience to the next level, adding NFC chips to figures, making the game more appeal to its users. 

Reviewer 3: ```suggestion
Skylanders is a game that uses NFC interaction between devices to get the user experience to the next level. By adding NFC chips to action figures, they make the game more appealing to its users. 
```

<img width="100%" src="/images/how-nfc-can-help-improve-your-business/skylanders-items.jpg" /> 
<br />
<p style="text-align: center;"> (source: <a href="https://www.theverge.com">The Verge</a>)</p>

> There are also a variety of consoles that have NFC integrated to them. In this cases some developers have come up with applications that could link devices to exchange game objects or share game play adding to it a competitive and cooperative experience. 

Reviewer 2: `...of consoles that already have NFC integrated.`

Reviewer 2: `In these cases, developers have come up with apps that can link devices in order to exchange in-game objects or to share gameplays, adding to a more competitive and cooperative experience, respectively.`

> This kinds of interactions help make the user experience more remarkable to the user and help make user interact more with the game itself. 

Reviewer 2: `These kind of interactions aid in making a more remarkable user experience, making the user interact even more with the game.`

> ### Interested in building NFC based applications get in touch!

Reviewer 2: `Interested in building NFC based applications? Get in touch!`

> There are also a variety of consoles that already have NFC integrated to them. In this cases some developers have come up with apps that can link devices in order to exchange in-game objects or share game play adding to it a more competitive and cooperative experience, respectively. 

Reviewer 3: ```suggestion
There are also a variety of consoles that already have NFC integrated to them. In this case, developers have come up with apps that link devices in order to exchange in-game objects or share game play adding to it a more competitive and cooperative experience, respectively. 
```

> A way to exchange information pair to pair securely and efficient, NFC based applications allows you to create mechanism to not only make yours business presses more secure but to improve their efficiency and help create a more appeal user experience. This technology is used in a variety of industries and have prove to provide an improvement to its owners.

Reviewer 2: `As a means to exchange...`

Reviewer 2: `...securely and efficiently, NFC based apps allow you to create a mechanism to not only make your business processes more secure and efficient but also to help create a more appealing user experience. This technology is used in a variety of industries and have proven to provide an improvement to its owners.`

> This kinds of interactions aid in making a more remarkable user experience, giving the user a way to interact even more with the game.

Reviewer 3: ```suggestion
This kind of interactions aid in making a more remarkable user experience, providing a way to engage with the game on a deeper level.
```

> To learn more about how NFC based apps can make improve your business, <a href="mailto:hi@xmartlabs.com">reach out to us today</a>.

Reviewer 2: `To learn more about how NFC based apps can improve your business...`

> As means to exchange information pair to pair securely and efficient, NFC based apps allows you to create mechanism to not only make yours business processes more secure and efficient but also to help create a more appeal user experience. This technology is used in a variety of industries and have proven to provide an improvement to its owners.

Reviewer 3: ```suggestion
As means to exchange information pair to pair securely and efficiently, NFC based apps allow you to create mechanisms to not only make your business processes more secure and efficient but also to help create a more appealing user experience. This technology is used in a variety of industries and has proven to provide clear benefits to its owners and users.
```
